LDAP/class.LDAPServer.php

```diff
@@ -104,12 +105,12 @@ public function connect($name, $proto = false)
         }
         $this->connect = $connectStr;
         $this->ldapLink = ldap_connect($this->connect);
+        ldap_set_option($this->ldapLink, LDAP_OPT_PROTOCOL_VERSION, 3);
```

Comment:
Get rid of this change and revert to the old way or we'll get "PHP Warning:  ldap_set_option(): supplied argument is not a valid ldap link resource" on bad connects. The false path shouldn't occur anyway...